views/_instructions.html

```diff
@@ -117,7 +123,7 @@
 
 <span class="hljs-comment"># Get resources</span>
 {% for resource in dataset.resources %}
-<span class="hljs-keyword">curl </span><span class="hljs-string">{{ resource.path }}</span>
+<span class="hljs-keyword">curl </span><span class="hljs-string">-L https://datahub.io/{{dataset.datahub.owner}}/{{dataset.name}}/r/{{loop.index0}}.{{resource.path.split('.').slice(-1)[0]}}</span>
```

Comment:
@AcckiyGerman is not this `{{resource.path.split('.').slice(-1)[0]}} `equivalent of `{{resource.format}}`?

Also why not `{{resource.name}}` instead of {{loop.index0}} ?

Reply:
@zelima I guess because `name` property can have suffixes such as `_csv` or `_json`. See discussion here - https://github.com/datahq/datahub-qa/issues/157#issuecomment-373279055

Reply:
@zelima agree about using `{{resource.format}}`